app/src/main/java/app/grapheneos/camera/CamConfig.kt

```diff
@@ -122,6 +124,8 @@ class CamConfig(private val mActivity: MainActivity) {
 
             val VIDEO_QUALITY = Quality.HIGHEST
 
+            val VIDEO_FRAME_RATE = Range<Int>(30, 30)
```

Comment:
Yes, this has been a requirement for basic recording for quite long, although we could adapt the code to handle situations where this might not be the case

Reply:
Have added a commit to handle cases when 30 fps is unavailable. Thanks!